pyatlan/model/assets/dbt_column_process.py

```diff
@@ -176,6 +176,10 @@ def __setattr__(self, name, value):
     """
     TBC
     """
+    ALTERYX_WORKFLOW: ClassVar[RelationField] = RelationField("alteryxWorkflow")
```

Comment:
This is a one-off customization (in-progress experiment) — we should remove from generated code.